pcsx2/GS/Renderers/Common/GSDevice.cpp

```diff
@@ -575,6 +579,22 @@ void GSDevice::DrawMultiStretchRects(
 	}
 }
 
+int GSDevice::SetDATMShader(u8& datm)
```

Comment:
This shouldn't be passed by reference. Pass it by value as a `SetDATM`, and cast it in the caller.

Function should also be either static, or outside of GSDevice.